<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inicio</class>
 <widget class="QMainWindow" name="Inicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="lblPesoIndicador">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>341</width>
         <height>111</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>26</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>-------------</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblEstadoIndicador">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>140</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Estado</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_restart">
       <property name="geometry">
        <rect>
         <x>840</x>
         <y>10</y>
         <width>171</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">#btn_restart {
    background-color: #F20000;
    color: rgb(255, 255, 255);
    border: none;
    border-radius: 5px;
    border-left: 1px solid #D10000;
    border-right: 1px solid #D10000;
    border-bottom: 5px solid #D10000;
}

#btn_restart_adc:hover {
    background-color: #D13600
}

#btn_restart_adc:pressed {
    background-color: #ED0000;
}
</string>
       </property>
       <property name="text">
        <string>REINICIAR CELDA</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_valor_min">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>16</y>
         <width>150</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_valor_max">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>120</y>
         <width>150</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbltxtvmin">
       <property name="geometry">
        <rect>
         <x>512</x>
         <y>20</y>
         <width>70</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>MIN :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lbltxtvmax">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>124</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>MAX :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_restart_cont">
       <property name="geometry">
        <rect>
         <x>840</x>
         <y>100</y>
         <width>171</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">#btn_restart_cont {
    background-color: #FF8000;
    color: rgb(255, 255, 255);
    border: none;
    border-radius: 5px;
    border-left: 1px solid #D96D00;
    border-right: 1px solid #D96D00;
    border-bottom: 5px solid #D96D00;
	justify-content: center;
 	align-items: center;
}

#btn_restart_adc:hover {
    background-color: #EB7900;
}

#btn_restart_adc:pressed {
    background-color: #FF8400;
}
</string>
       </property>
       <property name="text">
        <string>REINICIAR VALORES
Min/Max</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbltxtmargen">
       <property name="geometry">
        <rect>
         <x>508</x>
         <y>180</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>MARGEN :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_margen">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>180</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>- - - - - - -</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_peso_calibrado">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>331</width>
         <height>111</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>26</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>-------------</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_peso_real">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>370</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 5px;
border: 2px solid #DDD;
background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_calibrar">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>440</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">#btn_calibrar {
    background-color: #2F00ED;
    color: rgb(255, 255, 255);
    border: none;
    border-radius: 5px;
    border-left: 1px solid #2700C2;
    border-right: 1px solid #2700C2;
    border-bottom: 5px solid #2700C2;
}

#btn_restart_adc:hover {
    background-color: #4D00F2;
}

#btn_restart_adc:pressed {
    background-color: #4800E3;
}
</string>
       </property>
       <property name="text">
        <string>CALIBRAR</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_medio">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>70</y>
         <width>150</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbltxtmargen_2">
       <property name="geometry">
        <rect>
         <x>514</x>
         <y>72</y>
         <width>70</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ADC :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lbltxDisplay_PesoReal">
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>290</y>
         <width>231</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Colocar Peso Real 
en Kg</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_restart_adc">
       <property name="geometry">
        <rect>
         <x>840</x>
         <y>190</y>
         <width>171</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">#btn_restart_adc {
    background-color: #00D139;
    color: rgb(255, 255, 255);
    border: none;
    border-radius: 5px;
    border-left: 1px solid #089930;
    border-right: 1px solid #089930;
    border-bottom: 5px solid #089930;
}

#btn_restart_adc:hover {
    background-color: #20E355;
}

#btn_restart_adc:pressed {
    background-color: #14BC43;
}
</string>
       </property>
       <property name="text">
        <string>REINICIAR ADC</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_mostrar_peso">
       <property name="geometry">
        <rect>
         <x>830</x>
         <y>440</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">#btn_mostrar_peso {
    background-color: #00D139;
    color: rgb(255, 255, 255);
    border: none;
    border-radius: 5px;
    border-left: 1px solid #089930;
    border-right: 1px solid #089930;
    border-bottom: 5px solid #089930;
}

#btn_restart_adc:hover {
    background-color: #20E355;
}

#btn_restart_adc:pressed {
    background-color: #14BC43;
}</string>
       </property>
       <property name="text">
        <string>MOSTRAR EN KG</string>
       </property>
      </widget>
      <widget class="QComboBox" name="combo_Port">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>14</y>
         <width>100</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_port">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>10</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>AlphaMaleModern</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>PUERTOS</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lbltxKG">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>290</y>
         <width>101</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>KG</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <zorder>lblPesoIndicador</zorder>
      <zorder>lblEstadoIndicador</zorder>
      <zorder>btn_restart</zorder>
      <zorder>lbl_valor_min</zorder>
      <zorder>lbl_valor_max</zorder>
      <zorder>lbltxtvmin</zorder>
      <zorder>lbltxtvmax</zorder>
      <zorder>lbltxtmargen</zorder>
      <zorder>lbl_margen</zorder>
      <zorder>lbl_peso_calibrado</zorder>
      <zorder>txt_peso_real</zorder>
      <zorder>btn_calibrar</zorder>
      <zorder>lbl_medio</zorder>
      <zorder>lbltxtmargen_2</zorder>
      <zorder>lbltxDisplay_PesoReal</zorder>
      <zorder>btn_restart_adc</zorder>
      <zorder>btn_mostrar_peso</zorder>
      <zorder>combo_Port</zorder>
      <zorder>lbl_port</zorder>
      <zorder>lbltxKG</zorder>
      <zorder>btn_restart_cont</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
